src/main/target/EMAXF4SX1280/target.h

```diff
@@ -0,0 +1,131 @@
+#define BEEPER_PIN              PC15
+#define BEEPER_INVERTED
+
+#define ENABLE_DSHOT_DMAR       DSHOT_DMAR_AUTO
+#define DSHOT_BITBANG_DEFAULT   DSHOT_BITBANG_OFF
+
+// *************** SPI1 Gyro & ACC **********************
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define GYRO_1_CS_PIN           PA4
+#define GYRO_1_SPI_INSTANCE     SPI1
+
+#define USE_EXTI
+#define USE_GYRO_EXTI
+#define GYRO_1_EXTI_PIN         PA1
+#define USE_MPU_DATA_READY_SIGNAL
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM20689
+#define GYRO_1_ALIGN            CW180_DEG
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM20689
+#define USE_ACCGYRO_BMI270
+
+// *************** SPI2 OSD *****************************
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE    SPI2
+#define MAX7456_SPI_CS_PIN      PB12
+
+// *************** SPI3 RX ***************************
+#define USE_SPI_DEVICE_3
+#define RX_SPI_INSTANCE         SPI3
+#define SPI3_SCK_PIN            PB3
+#define SPI3_MISO_PIN           PB4
+#define SPI3_MOSI_PIN           PB5
+#define RX_NSS_PIN              PA15
+
+#define USE_RX_SPI
+#define USE_RX_EXPRESSLRS
+#define USE_RX_SX1280
+
+#define RX_SPI_LED_INVERTED
+#define RX_SPI_BIND_PIN         		PB2
+#define RX_SPI_EXTI_PIN             	PC14
+#define RX_SPI_LED_PIN              	PB9
+#define RX_EXPRESSLRS_SPI_RESET_PIN     PA8
+#define RX_EXPRESSLRS_SPI_BUSY_PIN      PA13
+#define RX_EXPRESSLRS_TIMER_INSTANCE    TIM3
+
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SPI
+#define RX_SPI_DEFAULT_PROTOCOL RX_SPI_EXPRESSLRS
+
+// *************** UART *****************************
+#define USE_VCP
+
+#define USE_UART1
+#define UART1_TX_PIN            PA9
+#define UART1_RX_PIN            PA10
+
+#define USE_UART2
+#define UART2_TX_PIN            PA2
+#define UART2_RX_PIN            PA3
+
+#define USE_SOFTSERIAL1
+
+#define SERIAL_PORT_COUNT       4
+
+// *************** ADC *****************************
+#define USE_ADC
+#define ADC_INSTANCE         ADC1  // Default added
+#define ADC1_DMA_OPT            0  // DMA 2 Stream 0 Channel 0 
+
+#define VBAT_ADC_PIN            PB0
+#define CURRENT_METER_ADC_PIN   PB1
+
+#define USE_ESCSERIAL
+
+#define DEFAULT_FEATURES                (FEATURE_OSD | FEATURE_TELEMETRY )
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+#define CURRENT_METER_SCALE_DEFAULT 179
+
+#define TARGET_IO_PORTA         0xffff
+#define TARGET_IO_PORTB         0xffff
+#define TARGET_IO_PORTC         0xffff
+#define TARGET_IO_PORTD         (BIT(2))
+
+#define USABLE_TIMER_CHANNEL_COUNT 9
+#define USED_TIMERS             ( TIM_N(1)|TIM_N(2)|TIM_N(4)|TIM_N(5)|TIM_N(9))
```

Comment:
Please add some space to operators for readability:
```suggestion
#define USED_TIMERS             ( TIM_N(1) | TIM_N(2) | TIM_N(4) | TIM_N(5) | TIM_N(9) )
```